package.json

```diff
@@ -9,8 +9,9 @@
   "dependencies": {
     "apsl-react-native-button": "2.4.2",
     "async": "1.5.2",
-    "colors": "1.1.2",
-    "crypto-js": "3.1.6",
+    "chance": "^1.0.3",
```

Comment:
I think may be better placed as a development dependency, as its only used in the populate script (development) and no need to have `colors` or `change` packaged into the GoodQuestion app.